src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,579 @@
+it's easy to accidentally create page content that doesn't quite fit within the specified viewport.
+For example, an image that is displayed at a width wider than the viewport
+can cause the viewport to scroll horizontally.
+You should adjust this content to fit within the width of the viewport,
+so that the user does not need to scroll horizontally.
+
+The [Content is not sized correctly for the viewport](/content-width/)
+Lighthouse audit can help you automate the process of detecting overflowing content.
+
+### Images {: #images }
+
+An image has fixed dimensions and if it is larger than the viewport will cause a scrollbar.
+A common way to deal with this problem is to give all images a `max-width` of `100%`.
+This will cause the image to shrink to fit the space it has,
+should the viewport size be smaller than the image.
+However because the `max-width`, rather than the `width` is `100%`,
+the image will not stretch larger than its natural size.
+It is generally safe to add the following to your stylesheet
+so that you will never have a problem with images causing a scrollbar.
+
+```css
+img {
+  max-width: 100%:
+  display: block;
+}
+```
+
+#### Add the dimensions of the image to the img element {: #image-dimensions }
+
+When using `max-width: 100%` you are overriding the natural dimensions of the image,
+however you should still use the `width` and `height` attributes on your `<img>` tag.
+This is because modern browsers will use this information to reserve space for the image
+before it loads in,
+this will help to avoid [layout shifts](/optimize-cls/) as content loads.
+
+### Layout {: #layout }
+
+Since screen dimensions and width in CSS pixels vary widely between devices
+(for example, between phones and tablets, and even between different phones),
+content should not rely on a particular viewport width to render well.
+
+In the past, this required setting elements used to create layout in percentages.
+In the example below, you can see a two-column layout with floated elements, sized using pixels.
+Once the viewport becomes smaller than the total width of the columns, we have to scroll horizontally
+to see the content.
+
+<figure class="w-figure">
+  <img src="./fixed-width-layout.jpg" alt="Screenshot of a two-column layout with most of the second column outside the viewport">
+  <figcaption class="w-figcaption">
+    A floated layout using pixels. <a href="https://layout-floats-px.glitch.me/">See this example on Glitch</a>.
+  </figcaption>
+</figure>
+
+By using percentages for the widths, the columns always remain a certain percentage of the container.
+This means that the columns become narrower, rather than creating a scrollbar.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/layout-floats-percent?path=README.md&previewSize=100"
+    title="layout-floats-percent on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
```

Comment:
I added you to https://glitch.com/@webdev so you should be able to give that team access to the Glitch now

I'm getting a little cognitive dissonance with our recommendations here because the Glitch is overflowing the "viewport" (of the embed). How do you think we should address this?

![image](https://user-images.githubusercontent.com/4713486/82074216-4af0b480-968f-11ea-8999-07c32ee5f66d.png)

Reply:
I've added them to the team, still having issues getting the logo to stay on some of them. Sometimes it works and sometimes not.

The width issue was just my error, I'd remixed the fixed width example and neglected to change the width to max-width on the container. Should be sorted now.